spring-cloud-kubernetes-config/src/main/java/org/springframework/cloud/kubernetes/config/reload/ConfigReloadAutoConfiguration.java

```diff
@@ -140,11 +142,8 @@ public ConfigurationUpdateStrategy configurationUpdateStrategy(
 		private static void wait(ConfigReloadProperties properties) {
 			final long waitMillis = ThreadLocalRandom.current()
 					.nextLong(properties.getMaxWaitForRestart().toMillis());
-			try {
-				Thread.sleep(waitMillis);
-			}
-			catch (InterruptedException ignored) {
```

Comment:
if you want to ignore `InterruptedException` (which is a terrible decision IMHO), then do not use `Thread:sleep`, at all; instead use a construct that does not throw a checked exception and is silent on problems, too.